src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,53 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
 
+    public String resultString(List<String> list) {
+        StringBuilder builder = new StringBuilder();
+        if (list.size() > 0) {
+            Integer buySum = 0;
+            Integer supplySum = 0;
```

Comment:
```suggestion
            int buySum = 0;
            int supplySum = 0;
```

мені здається int буде достатньо тут

Reply:
Done